src/test/org/apache/sqoop/hcat/HCatalogImportTest.java

```diff
@@ -770,6 +770,29 @@ public void testTableCreation() throws Exception {
       null, true, false);
   }
 
+  @Test
+  public void testExternalTableCreation() throws Exception {
```

Comment:
Could you please add more test cases? E.g. for when the table already exists and either exception is thrown or we drop it based on the usage of --drop-and-create-hcatalog-table option?

Reply:
One test case for drop and create and one negative is added.

Reply:
Thanks for adding new test cases!